azerothcore/azerothcore-wotlk
# fix(DB): Elwynn Forest ported from Trinitycore.

## Changes Proposed:
-  Ported: https://github.com/TrinityCore/TrinityCore/issues/18772

## Issues Addressed:

https://github.com/azerothcore/azerothcore-wotlk/issues/5654

## SOURCE:


## Tests Performed:

- I adapted several statements to AC and then ran them on the database to confirm errors.
- I saw the changes in the game.


## How to Test the Changes:


1. Run the sql on your database, then, go to the game and verify:
-- Mark (Little Boy Playing in Goldshire) -- BACKPORTED FROM UDB 4.3.4
-- Joshua (Little Boy Playing in Goldshire) -- BACKPORTED FROM UDB 4.3.4
-- Eastvale Peasant Carrying Wood (Closer) - Has Wood In Hand (Model 89 wood?) -- BACKPORTED FROM UDB 4.3.4
-- Guard Pathing From Stormwind to Goldshire
-- Catpurse Pathing Just Outside Northshire -- BACKPORTED FROM UDB 4.3.4
-- Catpurse Pathing at Bandit Hideout -- BACKPORTED FROM UDB 4.3.4
-- Stormwind Guard Pathing from Goldshire to East Bridge -- BACKPORTED FROM UDB 4.3.4
-- Kobold Miner Pathing in Fargodeep Mine -- BACKPORTED FROM UDB 4.3.4
-- Bandit Pathing at Brackwell Pumpkin Patch -- BACKPORTED FROM UDB 4.3.4
-- Eastvale Peasant Carrying Wood (Far One) - Does Not Have Wood In Hand -- BACKPORTED FROM UDB 4.3.4
-- Servant of Azora Pathing in Circle in Tower of Azora -- BACKPORTED FROM UDB 4.3.4
-- Murloc and Murloc Steamrunner Pathing Around Crystal Lake -- BACKPORTED FROM UDB 4.3.4
-- Eastvale Lumberjack in Eastvale Lumbermill -- Backported From UDB 4.3.4
-- Eastvale Peasant Carrying Wood (2 on Other Side) - Closer One -- BACKPORTED FROM UDB 4.3.4
-- Eastvale Peasant Carrying Wood (Lone One Near Road) - Does Not Have Wood In Hand -- BACKPORTED FROM UDB 4.3.4
-- Fix Spawn Locations For Some NPC
-- General Fix NPC Respawn Times - Existing Times Are Too Short For Almost All NPC

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
